CMakeLists.txt

```
@@ -120,7 +120,12 @@ if(DJINNI_WITH_JNI)
 
   # Do not use the host's jni.h on Android as it is provided automatically by the NDK
   if(NOT ANDROID)
+    # avoid error messages if awt is not found, what seems to be on some sytems sometimes the case
+    set(JAVA_AWT_LIBRARY NotNeeded)
+    set(JAVA_AWT_INCLUDE_PATH NotNeeded)
     find_package(JNI REQUIRED QUIET)
+    # since we are lazy using all JNI_INCLUDE_DIRS, instead of listing that we use, remove NotNeeded from all:-)
+    list(REMOVE_ITEM JNI_INCLUDE_DIRS NotNeeded)
     target_include_directories(djinni_support_lib PUBLIC "${JNI_INCLUDE_DIRS}")
```

This is what I mean: If we'd change this line to
```suggestion
    target_include_directories(djinni_support_lib PUBLIC "${JAVA_INCLUDE_PATH}")
```
we'd not need the `list(REMOVE_ITEM JNI_INCLUDE_DIRS NotNeeded)`. 

The CMake docs say 

> JAVA_INCLUDE_PATH the include path to jni.h

afaik `jni.h` is all we need, so I think it's better to just add this to the include directories instead of being lazy and including `JNI_INCLUDE_DIRS`? 🤔

---

yes yes, I understand, I am just slow, its late 

we need both, on some platforms there is a second include path